### DISCUSSION TITLE
Love the Plugin - Tested with Linux?

### DISCUSSION THREAD

**Discussion by aetherbird** [tefkah/zotero-night#177] [General]
Thank you so much for this.  The flatpak install is looking odd  around the menu bar on Debian Linux.  Any thoughts?  

![image](https://github.com/tefkah/zotero-night/assets/25832191/f3e309dc-05e3-4078-9585-12c002b06d7a)

**Comment by aetherbird** [comment]
I've switched to the version directly from Zotero and this was resolved!

**Reply by tefkah** [reply]
glad to hear it! it was kind of hard for me to test this plugin on linux, so the new zotero version is a lifesaver in that regard
